src/components/ChatView.vue

```diff
@@ -43,8 +43,12 @@
 			</div>
 		</transition>
 		<MessagesList
+			role="region"
+			:aria-label="t('spreed', 'Conversation messages')"
 			:token="token" />
-		<NewMessageForm />
+		<NewMessageForm
+			role="region"
+			:aria-label="t('spreed', 'Post message')" />
```

Comment:
Shouldn't this  go within the newmessageform component?

Reply:
not this one: see I've defined a "role=region" which means that screen readers will announce this section as a region.
with a screen reader you have the option to jump directly to a specific region of the screen from anywhere.

Reply:
seems I had a separate PR for the other missing aria-label, see https://github.com/nextcloud/spreed/pull/4648

Reply:
Just sayinh that maybe those 2 lines should go here? 
https://github.com/nextcloud/spreed/blob/1bce4b8bdf8ea820c513bef83afdebe124a705ff/src/components/NewMessageForm/NewMessageForm.vue#L24
Aren't they going to end up on the newmessageform root element anyway?

Reply:
yes and no

We need to keep the aria-label here so that the region itself has a name. The screen reader will announce it "Entering post message region" when tabbing into it, even if you tab onto the file upload button first so we should not set the region on the post message field. 

I agree that the field itself also needs an aria-label.

Reply:
I've cherry-picked my changes from the other PR here to make it more complete.

Please let me know whether you understood the region thing. I can demo it to you if you like.